### PR TITLE
[AIRFLOW-3475] Move ImportError out of models.py

### PR DESCRIPTION
**Pull request by BasPH**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-3475
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

Move the ImportError class to a separate file as part of [the big models.py refactor](https://issues.apache.org/jira/issues/?jql=text%20~%20%22Refactor%3A%20Move%20out%20of%20models.py%22%20AND%20reporter%20in%20(Fokko)) (/airflow/models/importerror.py).

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

File: airflow/models/importerror.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```
Comment (mik-laj) [inline airflow/models/importerror.py]: Maybe this file should be called `exception.py` and contain all the exceptions associated with model?

Comment (holdenk) [reply airflow/models/importerror.py]: `exception.py` makes me think of Python exceptions, and `airflow/exceptions.py` contains Python exceptions. How about `errors.py` and we could group errors/failures in there? Just a thought.

**Review by holdenk — COMMENTED** [review]
cc @Fokko who reported the JIRA issue :)

**Comment by BasPH** [comment]
Good point. I'll rename to `errors.py`.

@mik-laj There's one other error-class in models.py -> InvalidFernetToken. I was thinking of grouping that into a `fernet.py` script since there's a few fernet-related functions and classes in models.py. So for now, `errors.py` will only contain ImportError.

**Comment by Fokko** [comment]
@BasPH I've restarted the failed tests. Maybe do a rebase? It seems to fail on k8s.

File: airflow/models/importerror.py (hunk 1 above)
Comment (mik-laj) [reply airflow/models/importerror.py]: It's OK for me.

**Comment by BasPH** [comment]
@Fokko the tests were flaky for a few days, I re-ran and all succeeds now.

**Review by Fokko — APPROVED** [review]

**Pull request merged**